### ISSUE TITLE
Bug in ordinals division

### ISSUE THREAD

**Issue by MaximilianAlgehed** [ashinkarov/heh#6]
Hi, just dropping this here to let you know there is a bug in your implementation of ordinal `divmod`, `ordinals.ml:187` should probably be the ratio of the two different coefficients.

**Comment by ashinkarov** [comment]
I don't think so.  I guess this is just a funny way to say that  `c = 1`.  Otherwise, can you give a counter example where you think this code breaks?
